nixos/modules/services/x11/display-managers/lightdm.nix

```diff
@@ -71,8 +71,9 @@ in
   # here determines the default: later modules (if enable) are
   # preferred.
   imports = [
-    ./lightdm-greeters/gtk.nix
+    ./lightdm-greeters/enso-os.nix
     ./lightdm-greeters/mini.nix
+    ./lightdm-greeters/gtk.nix
```

Comment:
Not sure if I interpreted the comment here wrong, from what I understand, gtk.nix should always be on the bottom?

Reply:
With #44926, it doesn't seem to be a problem, so I'll just remove the reordering.